Tasks/VsTest/task.json

```diff
@@ -1,198 +1,260 @@
 {
     "id": "EF087383-EE5E-42C7-9A53-AB56C98420F9",
     "name": "VSTest",
-    "friendlyName": "Visual Studio Test",
+    "friendlyName": "Visual Studio Test Plus Plus",
     "description": "Run tests with Visual Studio test runner",
     "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=835764)",
     "category": "Test",
     "visibility": [
         "Build",
         "Release"
     ],
-    "runsOn": [
+	"runsOn": [
         "Agent",
         "MachineGroup"
     ],
     "author": "Microsoft Corporation",
```

Comment:
can we test upgrade scenario by creating a build definition with 2.11 task and then uploading this task and check if we queue build things are fine